Sources/Server/server.swift

```diff
@@ -1 +1,68 @@
+import Foundation
+import Socket
+import Requests
 
+public class Server {
+
+    var listeningSocket: Socket!
+    var clientSocket: Socket!
+    var readData = Data()
+    var parser: RequestParser
+
+    public init(parser: RequestParser) {
+        self.parser = parser
+    }
+
+    public func run() {
+        do {
+            try self.listeningSocket = Socket.create()
+            try self.listeningSocket.listen(on: 5000)
+            repeat {
+                try clientSocket = self.listeningSocket.acceptClientConnection()
+                let parsedIncomingRequest = parseRequest(socket: clientSocket)
+                let httpMethod = parsedIncomingRequest.returnMethod()
+
+                if (httpMethod == "GET" || httpMethod == "POST") {
+                    response200(socket: clientSocket)
+                } else {
+                    response404(socket: clientSocket)
+                }
+            } while true
+        } catch let error {
+            print (error.localizedDescription)
+        }
+    }
+
+    private func response200(socket: Socket) {
```

Comment:
Similar to how you created a Request object, you might want to create a Response object to handle all of the response logic.